build/bom/pom.xml

```diff
@@ -384,11 +384,6 @@
                 <artifactId>infinispan-anchored-keys</artifactId>
                 <version>${version.infinispan}</version>
             </dependency>
-            <dependency>
-                <groupId>org.infinispan</groupId>
-                <artifactId>infinispan-quarkus-embedded</artifactId>
```

Comment:
Any reason we are removing this?

Reply:
This is being removed here as this commit originally followed up on https://github.com/infinispan/infinispan/pull/13782

My preference would be to also include ^ in this PR, i.e. remove native server entirely.

Reply:
We should not do this kind of surgery on a stable branch.